### PR TITLE
Add layer settings library

### PR DESCRIPTION
**Pull request by christophe-lunarg**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -66,6 +66,10 @@ else()
     endif()
 endif()
 
+find_package(VulkanUtilityLibraries CONFIG REQUIRED QUIET)
+
+target_compile_definitions(Vulkan::LayerSettings INTERFACE VK_ENABLE_BETA_EXTENSIONS) # Enable beta Vulkan extensions
```
Comment (juan-lunarg) [inline CMakeLists.txt:71]: Is this needed? Shouldn't line 39 handle that?

Comment (christophe-lunarg) [reply CMakeLists.txt]: Removed, it seems to work so good for me \o/

**Review by jeremyg-lunarg — COMMENTED** [review]
LGTM but I want to make sure @juan-lunarg is good with the cmake parts, given that we had problems there when VVL did this change

**Comment by juan-lunarg** [comment]
> LGTM but I want to make sure @juan-lunarg is good with the cmake parts, given that we had problems there when VVL did this change

CMake parts look fine.

**Review by juan-lunarg — CHANGES_REQUESTED** [review]
Please rebase on main. ndk-build removed from this repo.

File: CMakeLists.txt [hunk 2]
```diff
@@ -66,6 +66,8 @@ else()
     endif()
 endif()
 
+find_package(VulkanUtilityLibraries CONFIG REQUIRED QUIET)
```
Comment (juan-lunarg) [inline CMakeLists.txt:69]:
```suggestion
find_package(VulkanUtilityLibraries CONFIG REQUIRED)
```

File: build-android/known_good.json
Comment (juan-lunarg) [inline build-android/known_good.json:1]: This file has been deleted 🎉

File: scripts/known_good.json [hunk 3]
```diff
@@ -8,6 +8,20 @@
             "install_dir": "Vulkan-Headers/build/install",
             "commit": "v1.3.261"
         },
+        {
+            "name": "Vulkan-Utility-Libraries",
+            "url": "https://github.com/KhronosGroup/Vulkan-Utility-Libraries.git",
+            "sub_dir": "Vulkan-Utility-Libraries",
+            "build_dir": "Vulkan-Utility-Libraries/build",
+            "install_dir": "Vulkan-Utility-Libraries/build/install",
+            "commit": "d414e86858986b0479324c706fb37a8bfb09490b",
```
Comment (juan-lunarg) [inline scripts/known_good.json:17]: The 1.3.262 tag should work

**Review by juan-lunarg — CHANGES_REQUESTED** [review]

File: build-android/known_good.json
Comment (christophe-lunarg) [reply build-android/known_good.json:1]: Yes!! :D

File: CMakeLists.txt (hunk 2 above)
Comment (christophe-lunarg) [reply CMakeLists.txt:69]: I don't think it's a good practice to make a "request change" with no explanation on the matter.

This said, I changed it.

File: scripts/known_good.json (hunk 3 above)
Comment (christophe-lunarg) [reply scripts/known_good.json:17]: Fixed

**Review by jeremyg-lunarg — APPROVED** [review]
LGTM. Juan's concern was addressed by rebasing, it looks like.

**Review by juan-lunarg — APPROVED** [review]

**Pull request merged**